src/main/java/gift/product/service/PostServiceV1.java

```diff
@@ -0,0 +1,47 @@
+package gift.product.service;
+
+import gift.domain.Product;
+import gift.global.exception.NotFoundProductException;
+import gift.product.dto.ProductCreateRequest;
+import gift.product.dto.ProductResponse;
+import gift.product.dto.ProductUpdateRequest;
+import gift.product.repository.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PostServiceV1 implements ProductService{
```

Comment:
```suggestion
public class ProductServiceV1 implements ProductService{
```
을 의도하신 것인지, 아니면 `PostServiceV1`으로 만드신 이유가 있는지 궁금합니다~

Reply:
헉 오타인것 같습니다 수정하겠습니다!